src/taipy/core/scenario/_scenario_manager.py

```diff
@@ -306,6 +306,7 @@ def _delete_by_version(cls, version_number: str):
             if scenario.cycle and len(cls._get_all_by_cycle(scenario.cycle)) == 1:
                 _CycleManagerFactory._build_manager()._delete(scenario.cycle.id)
             cls._repository._delete(scenario.id)
+            _publish_event(cls._EVENT_ENTITY_TYPE, scenario.id, EventOperation.DELETION, None)
```

Comment:
```suggestion
            super()._delete(scenario.id)
```
I think this would be cleaner

Reply:
Yes, indeed. But this should be refactored for the 3.0 version anyway.